ADAL/src/public/ADAuthenticationContext.h

```diff
@@ -280,12 +280,12 @@ typedef enum
     when needed, leveraging the parentController property.
  
  Note that on iOS and iPadOS devices it is recommended to configure WKWebView to use mobile content mode to guarantee consistent experience across all mobile apps.
+ Likewise, on MacOS, it is recommended to configure WKWebView to use destkop content mode.
  
+ New ADAL release has a new API "defaultWKWebviewConfiguration" that will return a default configuration object with recommended settings enabled for both iOS and MacOS.
```

Comment:
No need to talk about new ADAL release and new API here. You can just say that [ADAuthenticationParameters defaultWKWebviewConfiguration] can be used to get a preconfigured instance of WKWebViewConfiguration.